README.md

```diff
@@ -1 +1,34 @@
-# ecmascript2015-awesomebook
\ No newline at end of file
+## Project Name
+# ecmascript2015-awesomebook
+"Awesome Book refactor using es6 module"  is a project design to add and remove book from the web page, its a solo project set up by Microverse as a core requirement the code was broken into smaller fragments called module. 
+
+It is was redesign with javascript class, class method. 
+There are three (3) diferent sections that were added using event listener.
+if click on add, it will display the add books section
+if click on contact, it will display the contact information section
+if click on list, it will display the list of books sections
+
+## Built With
+The major used for this project are : 
+HTML,
+Javascript 
+
```

Comment:
- [ ] Kindly always add the Getting Started section in your ReadMes with steps to clone the repo since most recruiters are HR professionals and might not have the technical expertise to setup your repository for use,